.travis.yml

```diff
@@ -1,40 +1,46 @@
-os:
-  - linux
-  - osx
 language: generic
-sudo: required
-dist: trusty
-osx_image: xcode10.2
-services: docker
 
 before_install:
-  - if [ $TRAVIS_OS_NAME == "linux" ]; then
-      SWIFT_VERSION=$(<.swift-version);
-      docker pull swift:${SWIFT_VERSION};
-    elif [ $TRAVIS_OS_NAME == "osx" ]; then
-      rvm use 2.5.1 --install --binary --fuzzy;
-      ruby --version;
-      gem install bundler;
-      bundle --version;
-      brew update;
-      brew outdated swiftlint || brew upgrade swiftlint;
-    fi
+  - export SWIFT_VERSION=$(<.swift-version)
+  - export SWIFT_DOCKER_IMAGE=swift:${SWIFT_VERSION}-bionic
 
-install:
-  - if [ $TRAVIS_OS_NAME == "osx" ]; then
-      bundle install --jobs=3 --retry=3 --deployment;
-    fi
+stages:
+- name: test
+  if: tag IS NOT present
```

Comment:
Do we not want to deploy the changelings to Github when a tag is present?

Reply:
because this is an open source repo it wasn't initially setup for that

Reply:
Alright, and we don't want to add that in now?

Reply:
we don't change this that often ... next time ...